packages/clay-shared/src/FocusScope.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import React from 'react';
+import {useFocusManagement} from './useFocusManagement';
+
+interface IProps {
+	arrowKeys?: boolean;
+	focusManager: ReturnType<typeof useFocusManagement>;
+}
+
+const ARROW_DOWN_KEY_CODE = 40;
+const ARROW_UP_KEY_CODE = 38;
+const TAB_KEY_CODE = 9;
+
+export const FocusScope: React.FunctionComponent<IProps> = ({
```

Comment:
Maybe some overall description of the purpose for this component would be helpful here.

Reply:
Yeah, I'll write something, but just stressing that it's just for our internal use.